doc/kubernetes/modules/ROOT/pages/running/metrics/external_infinispan.adoc

```diff
@@ -0,0 +1,22 @@
+= External {jdgserver_name} deployment
+:description: This document contains details of the external {jdgserver_name} metrics that can be used to monitor your deployment's performance
+:jgrp_application: {jdgserver_name}
+:clustered_caches: true
+:cross_site: true
+
+{description}
+
+include::partial$/running/metrics/ispn_enable_metrics.adoc[leveloffset=+1]
+
+== Clustering and Network
+
+Multiple {project_name} nodes allows to distribute the load among them, but it requires communication between the nodes.
+This section describes the useful metrics to monitor the network between {project_name} nodes and possible network issues.
```

Comment:
"This section describes metrics that are useful for monitoring the communication between {project_name} in order to identify possible network issues."